com.serveftp.officefactoring.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: com.serveftp.officefactoring
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+rename-icon: officefactoring
+command: OfficeFactoring
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+  - /share/aclocal
+  - /man
+  - /share/man
+  - /share/gtk-doc
+  - '*.la'
+  - '*.a'
+  - /lib/girepository-1.0
+  - /share/doc
+  - /share/gir-1.0
+finish-args:
+  - '--socket=x11'
+  - '--share=ipc'
+  - '--share=network'
+  - '--filesystem=host'
+  - '--metadata=X-DConf=migrate-path=/org/gnome/dictionary/'
+modules:
+  - name: OfficeFactoring
+    buildsystem: simple
+    build-commands:
+      - install -D OfficeFactoring /app/bin/OfficeFactoring
+      - install -D libmidas.so.2 /app/bin/libmidas.so.2
+      - >-
+        install -D officefactoring.png
+        /app/share/icons/hicolor/256x256/apps/officefactoring.png
+      - >-
+        install -D com.serveftp.officefactoring.metainfo.xml
+        /app/share/metainfo/com.serveftp.officefactoring.metainfo.xml
+      - >-
+        install -D com.serveftp.officefactoring.desktop
+        /app/share/applications/com.serveftp.officefactoring.desktop
+    sources:
+        url: http://officefactoring.serveftp.com/demo/officefactoring-1.tar.gz
```

Comment:
as I mentionned previously you extract binaries that I suspect are x86_64, yet Flathub is also trying to build an aarch64 package as well.

Either you provide aarch64 binaries, or build from source, or see https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
Meu arquivo flathub.json poderia ser assim?

{
   "only-arches":[
      "x86_64"
   ],
   "skip-arches":[
      "i386"
   ]
}